src/test/java/com/cloudbees/jenkins/plugins/bitbucket/BranchScanningTest.java

```diff
@@ -66,13 +66,13 @@
     @Test
     public void uriResolverTest() throws Exception {
         BitbucketSCMSource source = getBitbucketSCMSourceMock(BitbucketRepositoryType.GIT);
-        String remote = source.getRemote("amuniz", "test", source.getRepositoryType());
+        String remote = source.getRemote(repoOwner, repoName, source.getRepositoryType());
```

Comment:
Yeah I dont see why Antonio would not have used his repoOwner variable and instead directly injected his ID and repo as a string

This seems like a good change to me @svanoort